data/data-pipeline/data_pipeline/etl/sources/tree_equity_score/etl.py

```diff
@@ -83,8 +83,52 @@ def transform(self) -> None:
             pd.concat(tes_state_dfs), crs=tes_state_dfs[0].crs
         )
 
+        # rename ID to Tract ID
+        self.df.rename(
+            columns={"geoid": ExtractTransformLoad.GEOID_FIELD_NAME},
+            inplace=True,
+        )
+
     def load(self) -> None:
         logger.info("Saving Tree Equity Score GeoJSON")
         # write nationwide csv
         self.CSV_PATH.mkdir(parents=True, exist_ok=True)
-        self.df.to_file(self.CSV_PATH / "tes_conus.geojson", driver="GeoJSON")
+        # here I remove the geometry
```

Comment:
To be nitpicky, could we clean up these comments a bit to be more readable to an external audience encountering this codebase for the first time? 

Sometimes your comments read like things that I would typically put as comments on the pull request as explanations of the current work and changes to collaborators, but those comments are ephemeral, not designed to last or make sense.

There's no "I" meaningfully in a codebase written by many authors, so I would avoid using personal pronouns.

Please use comments to be really clear and unambiguous about what you mean. I don't actually know what you mean by "remove the geometry" – do you mean something like "drop the fields related to geojson boundaries"? I  don't know.